iml-gui/crate/src/components/command_modal.rs

```diff
@@ -572,12 +577,36 @@ fn extract_uri_id<T: EndpointName>(input: &str) -> Option<u32> {
     })
 }
 
-fn is_finished(cmd: &RichCommand) -> bool {
-    cmd.complete
+fn to_load_cmd(model: &Model, cmd_id: u32) -> bool {
+    // load it if the command is not found or is not complete
```

Comment:
load what? This comment is confusing :)

Reply:
It = the command with the id `cmd_id`. If you want it to update, I can do this here: https://github.com/whamcloud/integrated-manager-for-lustre/pull/1899
(it sits on top of the current one)

Reply:
I would just not write any comments ;-)